public/manifest.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "BBC News",
   "short_name": "BBC News",
-  "Scope": "/",
+  "Scope": "/news/",
   "background_color": "#b80000",
   "display": "standalone",
-  "start_url": "/",
+  "start_url": "/news/",
```

Comment:
With this being hardcoded to a service such as `/news/` will we need a manifest per service once the WS sites are included?